### PR TITLE
Docker Java

### PR DESCRIPTION
**Pull request by fbrissi**
Create Docker to run Java applications:

- Open JDK alpine is avaliable in versions 7 and 8
- Version 9 to 11 is debian-slim
- Version later equals than 12 is oraclelinux7

### CODE REVIEW

**Comment by fabriciojs** [comment]
@fbrissi nice this looks good!

Just one question - since I am not familiar with Java development - do we really need/want to have all this versions available `[7, 8, 9, 10, 11, 12, 13, 14, 15, 16]`? Couldn't we narrow down this group? Just thinking that less images to maintain should benefit us in the long run.

**Comment by fbrissi** [comment]
@fabriciojs I think the OpenJDK was many confused, but it's an reflection the licence of JVM, I found an alternative to OpenJDK [Amazon Corretto](https://aws.amazon.com/pt/corretto/), I will change to this implementation.

https://github.com/corretto/corretto-docker

https://github.com/docker-library/openjdk/commit/3eb0351b208d739fac35345c85e3c6237c2114ec
https://github.com/docker-library/openjdk/commit/ef85d041fec079ae84f46d240d5580cd975632e4
https://github.com/docker-library/openjdk/commit/1308097ccaa6edc05ffc90b275185a69382a8be7
https://github.com/docker-library/openjdk/commit/2822f33cda71487d9e16500513f782c30a4dffbb
https://github.com/docker-library/openjdk/commit/485e574acc5d1922ff1778884aeecd82539edcbc

**Comment by fabriciojs** [comment]
Hey @fbrissi ! Just wanted to check in the status of this PR. We are waiting for the changes with the new base images, right?

**Comment by fbrissi** [comment]
Yes @fabriciojs. 
I need to do this. 
I think I'll be able to do that next week, we have an tag the WIP.

**Comment by fbrissi** [comment]
This is ok now.

🥳 🥳 🥳 🥳 🥳 🥳 🥳 🥳 


image

File: 11-prod/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+FROM amazoncorretto:11-alpine
+
+ENV ASUSER='' \
+    UID='' \
```
Comment (fabriciojs) [inline 11-prod/Dockerfile:4]: We should add in to the README (based on the standard format for our other image repos), all of these Environment Variables which the user may want to change when using it.

Comment (fbrissi) [reply 11-prod/Dockerfile:4]: Right

**Review by fabriciojs — APPROVED** [review]

**Pull request merged**